inst/compareScenarios/cs_01_stocks_flows.Rmd

```diff
@@ -0,0 +1,49 @@
+# Stock and flows
+
+## Building stock
+
+```{r stock vars}
+tot <- "Stock|Buildings"
+subsec <- c("Stock|Residential",
+            "Stock|Commercial")
+```
+
+```{r StockSubsector}
+showAreaAndBarPlots(data, vars = subsec, tot = tot, orderVars = "user", scales = "fixed")
+```
+
+### All buildings
+
+### Residential
+```{r residential stock vars}
+tot <- "Stock|Residential"
+heatingRes <- paste(tot, heating, sep = "|")
+vintageRes <- paste(tot, vintage, sep = "|")
+typeRes <- paste(tot, type, sep = "|")
+locationRes <- paste(tot, location, sep = "|")
+```
+
+#### by building type
+```{r Stock-Residential-(type)}
+showAreaAndBarPlots(data, vars = typeRes, tot = tot, orderVars = "user", scales = "fixed")
+walk(typeRes, showLinePlots, data = data)
+```
+
+#### by location
+```{r Stock-Residential-(location)}
+showAreaAndBarPlots(data, vars = locationRes, tot = tot, orderVars = "user", scales = "fixed")
+walk(locationRes, showLinePlots, data = data)
+```
+
+#### by construction cohort
+```{r Stock-Residential-(vintage)}
+showAreaAndBarPlots(data, vars = rev(vintageRes), tot = tot, orderVars = "user", scales = "fixed")
+```
```

Comment:
Yes, it was. I thought that they are not very insightful as you'll just have decaying quantities by design where the trend won't give you much information. Might be that one can find it helpful when comparing scenarios against each other. If you think it could be helpful, I can add it of course.

Reply:
No, if you say it's not helpful, we don't need it just now. We can still add it later.